cluster-provision/centos9/vmcli/cmd/qemu/system.go

```diff
@@ -0,0 +1,78 @@
+package qemu
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/google/uuid"
+)
+
+// The name of the QEMU main executable
+const qemuExec = "qemu-system-%s"
```

Comment:
This script is actually running in a [Fedora 39](https://github.com/kubevirt/kubevirtci/blob/d69cd4e5c4591a8275a176cb5aa9b0b2fb53bcd1/cluster-provision/centos9/Dockerfile#L2) container, so this is why it currently works.

Do we want to add a check for the current OS and use `qemu-kvm` if available instead?

Reply:
Ah I see. I think we should get rid of fedora in the future. It is up to you, either you can a put a comment with a TODO and we can address this in a second PR or you can add it here.

Reply:
I think it'd be better to address this in a second PR, so this one doesn't get too big

Reply:
Agreww

Reply:
Why do we need to get rid of Fedora in this context?

Reply:
we don't need to. But we have converted everything to centos stream. It seems a bit inconsistent to me that we still keep fedora. Aynway, this can be a topic for another PR.